### ISSUE TITLE
Creating window with more than one url leads to error

### ISSUE THREAD

**Issue by TheUnsleepingAlchemist** [extension-js/extension.js#112]
both will leads to error:
```js
chrome.windows.create({url:["https://www.google.com/"]})
```
```js
chrome.windows.create({url:["https://www.google.com/","https://www.google.com/"]})
```

![image](https://github.com/extension-js/extension.js/assets/46828559/c1794736-a114-4998-86c4-7781b5fbb103)

error doesn't show up while trying to create a window with string

this one will work fine:
```js
chrome.windows.create({url:"https://www.google.com/"})
```

I used version 1.7.0 with --template=chatgpt

**Comment by cezaraugusto** [comment]
Hi @TheUnsleepingAlchemist thanks a lot for reporting!

I created #117 to address this one

**Comment by cezaraugusto** [comment]
@TheUnsleepingAlchemist fixed in extension@1.8.0, please update!

I'm closing this one, please feel free to open it again in case I'm wrong. Thanks!
